core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java

```diff
@@ -0,0 +1,28 @@
+package org.polarsys.capella.core.semantic.queries.basic.queries;
```

Comment:
Can you add copyright headers like other files on new files
https://github.com/eclipse/capella/wiki/How-to-contribute#license-headers

Reply:
done